AdaptiveCards/templating/sdk.md

```diff
@@ -85,32 +85,113 @@ dotnet add package AdaptiveCards.Templating --version 0.1.0-alpha1
 >
 > Consider changing the version above to the latest published version
 
+### Usage
 Import the library 
 
 ```cs
 using AdaptiveCards.Templating
 ```
 
-Use the templating engine by passing in your template JSON and data JSON.
-
 ```cs
 var templateJson = @"
 {
     ""type"": ""AdaptiveCard"",
-    ""version"": ""1.0"",
+    ""version"": ""1.2"",
     ""body"": [
         {
             ""type"": ""TextBlock"",
-            ""text"": ""Hello {name}""
+            ""text"": ""Hello ${name}!""
         }
     ]
 }";
 
-var dataJson = @"
-{
-    ""name"": ""Mickey Mouse""
+var dataJson = @"{
+    ""person"": { 
+        ""name"": ""Mickey Mouse""
```

Comment:
should we change this to avoid any kind of copyright complain?

Reply:
There is a similar one for the JS one on #274